assets/js/components/ProjectEditor.js

```diff
@@ -103,15 +120,17 @@ export function ProjectEditor (projectDescriptionCredits, programId, textFields,
 
   this.close = () => {
     $(window).off('popstate', this.popStateHandler)
-    showDefaultTopBarTitle()
+    $(window).on('popstate', this.navigationHandler)
 
     if (this.reloadScreen) {
       window.location.reload()
     }
 
-    this.body.removeClass('overflow-hidden')
     this.editTextUI.addClass('d-none')
+    this.editTextNavigation.removeClass('d-none')
```

Comment:
same here, Should this be in reopenNavigation or does it break things?

Reply:
Moved to reopenNavigation